slices/debianutils.yaml

```diff
@@ -0,0 +1,35 @@
+package: debianutils
+
+essential:
+  - debianutils_copyright
+
+slices:
+  bins:
+    essential:
+      - debianutils_config
+      - libc6_libs
+    contents:
+      /usr/bin/ischroot:
+      /usr/bin/run-parts:
+      /usr/bin/savelog:
```

Comment:
A few of these executables are scripts. We need to figure out how we are going to link /bin/sh in #337 first